src/server/Adaptive.ReactiveTrader.Server.Bot/src/messages/index.ts

```diff
@@ -0,0 +1,4 @@
+export {createPriceMessage} from './priceQuoteMessage'
```

Comment:
Minor: Spacing inconsistent here. Is there prettier config/editor config for stuff like this?